src/Microsoft.DotNet.Darc/Darc/Models/PopUps/AddSubscriptionPopUp.cs

```diff
@@ -61,7 +61,7 @@ public AddSubscriptionPopUp(
                 new("They define which directory of the VMR (under src/) are the sources synchronized with.", true),
                 new("Only one of those needs to be set based on whether the source or the target repo is the VMR.", true),
                 Line.Empty,
-                new("Excluded assets is a list of package names to be ignored during source-enabled subscriptions (VMR code flow). ", true),
+                new("Excluded assets is a list of package names to be ignored during dependency updates. ", true),
```

Comment:
Probably excluded assets should be moved up (before source and target directories) in the popups so that it's clear they don't belong to that section.